images

```diff
@@ -0,0 +1 @@
+content/images
```

Comment:
What changed here?  We should revert this change so that the build doesn't break.